layouts/partner-form/single.html

```diff
@@ -54,23 +79,17 @@
           // if email input does not include ampersand
           if(userEmailInput && userEmailInput.value && !userEmailInput.value.includes("@")) {
             errElm[index].setAttribute("style", "display:block;");
-          } else {
-            // next button successful click logic
-            parent[index].setAttribute("style", "display:none;");
-            if(backButton) {
-              backButton.setAttribute("style", "display:inline-block;");
-            }
-            if(progressBar) {
-              progressBar.innerHTML = index + 2
-              progressBarValue = progressBarValue + 8.333;
-              progressBar.setAttribute("style", `width: ${progressBarValue}%`);
-            }
-            index = index + 1
-            parent[index].setAttribute("style", "display:block;");
-            if(parent[index].classList.contains('partner-form-per-month')) {
-              nextButton.setAttribute("style", "display:none;");
-              submitButton.setAttribute("style", "display:inline-block;");
-            }
+          } else if(parent[index].classList.contains("partner-form-use-case")) {
+						var selected = false;
+						for (var i = 0, length = useCaseOptions.length; i < length; i++) {
+		          if (useCaseOptions[i].selected) {
+								selected = true
+								successfulClick()
+		          }
+		        }
+						!selected && errElm[index].setAttribute("style", "display:block;");
+					} else {
```

Comment:
do we need this else?

or we could call successfulClick without else?

Reply:
I think the else is needed because it should not be called in certain scenarios